src/AndcultureCode.CSharp.Sitefinity.Core/Services/Blogs/CoreBlogODataServices.cs

```diff
@@ -0,0 +1,16 @@
+﻿using AndcultureCode.CSharp.Sitefinity.Core.Interfaces;
+using AndcultureCode.CSharp.Sitefinity.Core.Models.Configuration;
+
+namespace AndcultureCode.CSharp.Sitefinity.Core.Services.Blogs
+{
+    /// <summary>
+    /// Represents the out of the box (OOB) Sitefinity OData blog service available
```

Comment:
Should we consider just dropping `(OOB)` from all these summary comments?  Seems like unnecessary noise repeated.